src/sentry/static/sentry/app/components/commitLink.tsx

```diff
@@ -81,27 +79,10 @@ function CommitLink({inline, commitId, repository}: Props) {
     </Button>
   ) : (
     <ExternalLink className="inline-commit" href={commitUrl}>
-      <CommitIcon src={providerData.icon} />
+      {providerData.icon}
```

Comment:
@doralchan Since `providerData.icon` is a REact component, this looks kind of funny to me. Is it possible to do something like:
```
<providerData.icon/>
```

Reply:
Is it? I haven't tried anything like that before o_O

Reply:
![Screen Shot 2020-03-09 at 5 10 53 PM](https://user-images.githubusercontent.com/4830259/76267744-00832f80-6229-11ea-9827-0d7e14d6890a.png)

This generates an error that says it `does not have any construct or call signatures` 

Is there something I should be changing?

Reply:
@doralchan If my suggestion doesn't work, don't sweat it. What you have is fine I think.